src/main/java/org/jboss/windup/plugin/WindupMojo.java

```diff
@@ -73,7 +79,7 @@
     /**
      * Location of the input file application.
      */
-    @Parameter(alias = "input", property = "input", required = true)
+    @Parameter( defaultValue = "${project.build.sourceDirectory}", property = "input", required = true )
```

Comment:
Unless you also change the quickstart to "inputDirectory", dropping the alias will break it.